geoportailv3/models.py

```diff
@@ -77,3 +77,13 @@ class LuxGetfeatureDefinition(Base):
     attributes_to_remove = Column(Unicode, label=_(u'Attributes to keep'))
     poi_id_collection = Column(Unicode, label=_(u'Id of the poi collection'))
     geometry_column = Column(Unicode, label=_(u'Geometry column name'))
+
+
+class LuxPrintJob(Base):
+    __tablename__ = 'lux_print_job'
+    __table_args__ = {'schema': _schema}
+
+    id = Column(Integer, primary_key=True)
+    ref = Column(String, index=True, unique=True, nullable=False)
+    spec = Column(Unicode)
```

Comment:
I never use JSON column  type, then I just have some quick questions:
What the advantages? Type verification? Shouldn't we passed the JSON on the read and on the write than it will takes some time?